sematic/ui/packages/common/src/pages/RunDetails/logs/BidirectionalLogView.tsx

```diff
@@ -141,22 +143,7 @@ const BidirectionalLogView = (props: BidirectionalLogViewProps) => {
             </Alert>}
 
             {lines.map((line, index) => (
-                <Box
-                    sx={{
-                        borderTop: 1,
-                        borderColor: theme.palette.grey[200],
-                        fontSize: 12,
-                        py: 2,
-                        pl: 1,
-                        color: theme.palette.grey[800],
-                        backgroundColor:
-                            index % 2 === 0 ? "white" : theme.palette.grey[50],
-                        paddingRight: 1
-                    }}
-                    key={index}
-                >
-                    <code>{line}</code>
-                </Box>
+                <LineTemplate key={index} index={index} line={line} />
```

Comment:
I think we may want to recover the light border between lines, and potentially also the alternate background color for easier reading.

Reply:
Yes, it now looks like this:

<img width="1368" alt="image" src="https://github.com/sematic-ai/sematic/assets/133257643/a047eb0c-a911-4a86-ab7e-5d798a08d85c">